apiserver/facades/client/action/action_test.go

```diff
@@ -102,26 +101,18 @@ func (s *actionSuite) SetUpTest(c *gc.C) {
 	s.AddCleanup(func(_ *gc.C) { s.resources.StopAll() })
 }
 
-func (s *actionSuite) AssertBlocked(c *gc.C, err error, msg string) {
-	c.Assert(params.IsCodeOperationBlocked(err), jc.IsTrue, gc.Commentf("error: %#v", err))
-	c.Assert(errors.Cause(err), gc.DeepEquals, &params.Error{
-		Message: msg,
-		Code:    "operation is blocked",
-	})
-}
-
 func (s *actionSuite) TestBlockEnqueue(c *gc.C) {
 	// block all changes
 	s.BlockAllChanges(c, "Enqueue")
 	_, err := s.action.Enqueue(params.Actions{})
-	s.AssertBlocked(c, err, "Enqueue")
+	c.Assert(err, jc.ErrorIsNil)
```

Comment:
Strictly speaking, you should actually test that the Enqueue has happened - this test as written will pass but does not guarantee correct behaviour. You can simply add the BlockAllChanges() call to the existing Enqueue test and delete this test. Same with the other test

Reply:
Makes perfect sense.